indico/web/templates/forms/permissions_widget.html

```diff
@@ -24,6 +24,8 @@ <h3>Permissions</h3>
                         </a>
                     </li>
                 </ul>
+                <a class="i-button arrow js-dropdown" data-toggle="dropdown">{% trans %}Category role{% endtrans %}</a>
+                <ul class="i-dropdown entry-category-role-dropdown" data-items='{{ field.category_roles|tojson|forceescape }}'></ul>
```

Comment:
this should only be rendered if there are actually some roles (showing an empty dropdown is weird)